TreeDB/mongo_gateway/server_test.go

```
@@ -5751,6 +5792,201 @@ func TestCompareRawNumbersHandlesNonFiniteDoubles(t *testing.T) {
+}
+
+func wideRawDocumentValue(width int, final bool) bson.RawValue {
+	doc := make([]byte, 4)
+	for i := range width {
+		value := true
+		if i == width-1 {
+			value = final
+		}
+		doc = bsoncore.AppendBooleanElement(doc, strconv.Itoa(i), value)
+	}
+	doc = append(doc, 0)
```

golangci-lint (2.12.2)

[error] append to slice `doc` with non-zero initialized length

(makezero)